.platform/services.yaml

```diff
@@ -0,0 +1,11 @@
+# The services of the project.
+#
+# Each service listed will be deployed
+# to power your Platform.sh project.
+
+db:
+    type: mariadb:10.4
+    disk: 2048
+
+cache:
+    type: redis:6.0
```

Comment:
suggestion (non-blocking): There were discussions about changing `redis` to `redis-persistent` in all Drupal templates, what do you think? See https://platformsh.slack.com/archives/C022N5Q5LDP/p1665998074760619

Reply:
I'm not personally convinced that `redis/persistant` is the root cause or solution of the reported issue - that has not been shown except as an anecdotal suggestion. 
IMO, Drupal+redis cache usage should have no expectation of persistance https://platformsh.slack.com/archives/C0JBB9B4G/p1666704429441319?thread_ts=1666693323.220269&cid=C0JBB9B4G 

But either way - if this were to be agreed as a Drupal template change, then that template behaviour change should be applied to `platformsh-templates/drupal9` first, and that patch could then be forward-ported to here cleanly. The repo and branch now shares history with `platformsh-templates/drupal9` which means that patches for `platformsh-templates/drupal9` can (should) be able to be ported forwards to here.
This is actually a big deal, as anything done from now on in in `platformsh-templates/drupal9` can now be cherry-picked or merged directly into this repo. Could not do that before, and it would be a process streamlining.

I didn't want to fork any of the other misc `platformsh-templates/drupal9` cleanups into this govcms task, just get things going as-is. Global Drupal improvements need to start higher up the tree, and we can inherit it from there. I have some ddev and cache tweaks in mind also.